client/my-sites/email/email-providers-in-depth-comparison/style.scss

```diff
@@ -0,0 +1,5 @@
+.email-providers-in-depth-comparison__header {
+	font-size: xxx-large;
```

Comment:
Nit: should this font size not match the header size for the comparison page? (I have another comment where the current size of the comparison header is equivalent to `$font-size-large`.)

Reply:
Totally makes sense, I'll change that 👍.